internal/cluster/validator.go

```diff
@@ -406,7 +406,9 @@ func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) va
 			v.log.WithError(err).Warnf("Illegal inventory for host %s", h.ID.String())
 			continue
 		}
-		if inventory.Timestamp == 0 || *h.Status == models.HostStatusDisconnected || *h.Status == models.HostStatusDisabled {
+		if inventory.Timestamp == 0 || *h.Status == models.HostStatusDisconnected ||
+			*h.Status == models.HostStatusDisabled || *h.Status == models.HostStatusResettingPendingUserAction ||
+			*h.Status == models.HostStatusDiscovering {
```

Comment:
why you added discovering? I think it's wrong - you should validate it in all states that we are running inventory commands: https://github.com/openshift/assisted-service/blob/master/internal/host/instructionmanager.go#L75

Reply:
discussed and agreed it is ok